src/src/Helper/ShellHelper.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Helper;
+
+use App\Entity\Option;
+use App\Factory\TaskFactory;
+use App\Repository\OptionRepository;
+use Doctrine\ORM\EntityManagerInterface;
+
+class ShellHelper
+{
+    private OptionRepository $optionRepository;
+    private EntityManagerInterface $entityManager;
+    private TaskFactory $taskFactory;
+
+    public function __construct(
+        OptionRepository $optionRepository,
+        EntityManagerInterface $entityManager,
+        TaskFactory $taskFactory
+    ) {
+        $this->optionRepository = $optionRepository;
+        $this->entityManager = $entityManager;
+        $this->taskFactory = $taskFactory;
+    }
+
+    private function setOptionValue(string $name, string $value): void
+    {
+        $option = $this->optionRepository->findOneBy(['name' => $name]) ?? new Option();
+        $option->setName($name);
+        $option->setValue($value);
+        $this->entityManager->persist($option);
+        $this->entityManager->flush();
+    }
+
+    public function stopShell(): array
+    {
+        $task = $this->taskFactory->createStopShellTask();
+        $this->entityManager->persist($task);
+        $this->entityManager->flush();
+
+        return [
+            'task_id' => $task->getId(),
+            'task_status' => $task->getStatus(),
+            'task_args' => $task->getArgs(),
+        ];
+    }
+
+    public function startShell(int $timeout): array
+    {
+        $task = $this->taskFactory->createStartShellTask($timeout);
+        $this->entityManager->persist($task);
+        $this->entityManager->flush();
+
+        return [
+            'task_id' => $task->getId(),
+            'task_status' => $task->getStatus(),
+            'task_args' => $task->getArgs(),
+        ];
+    }
+
+    public function getShellStatus(): array
+    {
+        $shell_status_option = $this->optionRepository->findOneBy(['name' => 'SHELL_STATUS']) ?? new Option();
```

Comment:
Could be added to the OptionRepository

```php
    public function findShellStatus): ?string
    {
        return $this->findOption('SHELL_STATUS');
    }
```

Reply:
Yes, `findOption` already exists in `OptionRepository` :) 
I added repo methods to fetch these options.